docs/docs/service-discovery-load-balancing.md

```diff
@@ -55,3 +60,20 @@ $ ./bin/haproxy-marathon-bridge install_haproxy_system localhost:8080
 
 The provided script is just a basic example. For a full list of options, check the
 [HAProxy configuration docs](http://cbonte.github.io/haproxy-dconv/configuration-1.5.html).
+
+### servicerouter.py
+To generate an HAProxy configuration from Marathon running at `localhost:8080` with the `servicerouter.py` script:
+
+``` console
+$ ./bin/servicerouter.py --marathon http://localhost:8080 --haproxy-config /etc/haproxy/haproxy.cfg
+```
+
+This will generate a new haproxy.cfg and if it changed automatically reload a running haproxy.
+
+Servicerouter has a lot of additional functionality like sticky sessions, http to https redirection, SSL offloading,
```

Comment:
To be more consistent with the section name, I'd write:

servicerouter.py has a lot of additional functionality like sticky sessions, HTTP to HTTPS redirection, SSL offloading,

I also made the protocol names upper-case.